perfmetrics/scripts/continuous_test/gcp_ubuntu/continuous.cfg

```diff
@@ -18,6 +18,7 @@ action {
     regex: "gcsfuse-logs-ls-hns.txt"
     regex: "gcsfuse-logs-fio-flat.txt"
     regex: "gcsfuse-logs-ls-flat.txt"
+    regex: "gcsfuse-logs-rename-hns.txt"
```

Comment:
If these really are regex patterns, then these can be made more performant by

```suggestion
    regex: "gcsfuse-logs-rename-hns\.txt"
```

same goes for the other lines before this.

Reply:
Updated.